omniscidb/scripts/mapd-deps-conda-dev-env.yml

```diff
@@ -49,4 +50,5 @@ dependencies:
   - binutils
   - llvm-spirv 14.0.*
   - folly 2022.11.07.00
+  - dpcpp_cpp_rt
```

Comment:
Do we have to add this as a dependency for all environments? What does this do to our conda solve?

Reply:
Since `DwarfBench` support have been made optimal in CMake, and dpc++ runtime is used only there, it would probably be more correct to take out `dpcpp_cpp_rt`. How could I do it more correctly?

Reply:
@alexbaden, dwarf bench will require dpc++ runtime for all platforms. What happens to conda solve we need to find out :)

Reply:
removed